core/src/main/resources/lib/form/class-entry.jelly

```diff
@@ -34,7 +34,7 @@ THE SOFTWARE.
       structured form submission.  Mutually exclusive with clazz.
     </st:attribute>
   </st:documentation>
-  <j:set var="clazz" value="${attrs.clazz ?: attrs.descriptor.clazz.name}" />
+  <j:set var="clazz" value="${attrs.clazz ?: attrs.descriptor.id}" />
```

Comment:
Should adjust documentation for `clazz` to emphasize that it should be `Descriptor.id` more generally.